### PR TITLE
Pong: Init new Pong info

### PR DESCRIPTION
**Pull request by Ghosuto**

### CODE REVIEW

File: Pong.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 			"gapps": "https://nikgapps.com/crdroid-official",
 			"firmware": "",
 			"modem": "",
 			"bootloader": "",
-			"recovery": "https://sourceforge.net/projects/crdroid/files/Pong/10.x/recovery/",
-			"paypal": "https://www.paypal.com/paypalme/dylanakp",
-			"telegram": "https://t.me/+bsJmd23ZEy1jMTg0",
+			"recovery": 
+			"paypal": ""
,
```
Comment (gwolf2u) [inline Pong.json]: huh?

File: Pong.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 			"oem": "Nothing",
 			"device": "Nothing Phone (2)",
-			"filename": "crDroidAndroid-14.0-20240512-Pong-v10.5.zip",
-			"download": "https://sourceforge.net/projects/crdroid/files/Pong/10.x/crDroidAndroid-14.0-20240512-Pong-v10.5.zip/download",
-			"timestamp": 1715529588,
-			"md5": "cf9618d5e9c8b24c5f79012d84c47fe8",
-			"sha256": "5ede7ac4f7614e58802b6ab977f8c8651386b1653b4b135a42172b00b5a808b2",
-			"size": 1873273466,
-			"version": "10.5",
+			"filename": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+			"download": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
+			"timestamp": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+			"md5": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+			"sha256": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 7]
```diff
@@ -1,25 +1,25 @@
...
+			"size": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

File: Pong.json [hunk 8]
```diff
@@ -1,25 +1,25 @@
...
+			"version": "",
```
Comment (gwolf2u) [inline Pong.json]: mandatory

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by Ghosuto** [comment]
Updated

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**